### DISCUSSION TITLE
Getting "Connection errored out" every 15 minutes in Colab (^C in console)

### DISCUSSION THREAD

**Discussion by AndreiLungu64** [lllyasviel/Fooocus#1741] [General]
When I run Fooocus in Colab, it consistently crashes every 15 minutes, and I don't know what to do. I've observed that the RAM usage increases to 12.2 GB (out of a total of 12.7 GB), and within the next 1-2 minutes, it crashes. Besides upgrading to Colab Pro, is there a way to fix this issue? Can I somehow lower the RAM usage with an argument like '-lowram' or something?

**Comment by mashb1t** [comment]
see https://github.com/lllyasviel/Fooocus/issues/1756

**Reply by AndreiLungu64** [reply]
Thanks!
